benchmark/vectors.py

```diff
@@ -0,0 +1,20 @@
+from torchtext import vocab
+from torchtext import experimental
+from torchtext.data.utils import get_tokenizer
+import time
+
+
+def benchmark_fasttext():
```

Comment:
I'd remove the benchmark for vector because it will be a separate PR for the vector work.